jazz_environments/index.js

```diff
@@ -110,15 +110,14 @@ module.exports.handler = (event, context, cb) => {
         }
 
         // throw bad request error if user is unauthorized for GET
-        if (event.principalId === undefined || event.principalId === "" || event.principalId === null) {
+        if (event.principalId === undefined || !event.principalId) {
```

Comment:
don't need a check for undefined since you are already checking for `!event.principalId `

Reply:
Done